modules/library/metadata/src/main/java/org/geotools/util/factory/GeoTools.java

```diff
@@ -186,6 +186,26 @@ private static Properties loadProperites(String resource) {
     public static final String FORCE_LONGITUDE_FIRST_AXIS_ORDER =
             "org.geotools.referencing.forceXY";
 
+    /**
+     * The {@linkplain System#getProperty(String) system property} key for the default value to be
+     * assigned to the {@link Hints# ENTITY_RESOLVER} hint.
+     *
+     * <p>GML Encoding in geotools forces http://www.opengis.net/gml/srs/epsg.xml# syntax on CRS
+     * that are East\North oriented or have no orientation. This property prevents this behaviour.
```

Comment:
The ticket talks about one-dimensional CRS but the implementation seems to cover 2d CRSs as well.
Is the flag compatible with what declared at https://docs.geoserver.org/2.22.x/en/user/services/wfs/axis_order.html ?